search/base/inverted_list.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "base/assert.hpp"
+
+#include <algorithm>
+#include <cstddef>
+#include <vector>
+
+namespace search
+{
+namespace base
+{
+template <typename Id>
+class InvertedList
```

Comment:
Please add a comment, "inverted list" makes no sense to me.
Is it a "list that is intended to be used as an inverted index"? In an inverted index?
The implementation looks merely like a sorted list, why not call it that?

Reply:
Values inside inverted list may be sorted in any order, and the user of the class should not depend on this implementation detail. I added a comment to the class.